examples/grafana/grafana-ocp.yaml

```diff
@@ -10,13 +10,40 @@ metadata:
     iconClass: icon-cogs
     tags: "metrics,monitoring,grafana,prometheus"
 parameters:
+- description: The location of the proxy image
+  name: IMAGE_GF
+  value: mrsiano/grafana-ocp:latest
```

Comment:
@smarterclayton can we push this image under `openshift`? Also, the `-ocp` suffix seems redundant once we do that.

Reply:
make sense, bear in mind in the next future we'll use the latest grafna image, regarding the merged PR here https://github.com/grafana/grafana/pull/9504/, we don't have to use (patched grafana).

advantages:
use latest \ or last stable build.
we dont have to bump the version
we can use Datasources as configuration instead of patched grafan.

Reply:
If we need to use a grafana release that is yet to be released, I am fine with continuing to use your image for now. Evenutally, once the new released image is out, we probably want to push it in `docker.io/openshift`. Can you open an issue in this repo to track using the official image?